en/firefox/browsers/mobile/compare.ftl

```diff
@@ -0,0 +1,64 @@
+mobile-compare-almost-all-of = Almost all of the mobile browsers compared here allow complete synchronization between mobile and desktop devices.
+mobile-compare-the-good-news = The good news for { -brand-name-iphone } users is that { -brand-name-apple } has recently allowed { -brand-name-iphone } users to choose their own default browser, so now users can sync their devices across platforms if, for example, they install { -brand-name-firefox } on their { -brand-name-iphone } and { -brand-name-windows } PC.
+
+mobile-compare-conclusion = Conclusion
+mobile-compare-based-on-the = Based on the criteria we outlined — privacy, features, and syncing — there’s really only one mobile browser that satisfies all three, and that’s { -brand-name-firefox }. While similar in the features and syncing categories, { -brand-name-firefox } stands apart in privacy, because we feel it’s the essential component that allows you to just enjoy the internet — exactly the way you like it.
+mobile-compare-our-recommendations-aside = Our recommendations aside, finding the browser that’s right for you will always depend on your individual needs and what you value most as you navigate online.
+
+mobile-compare-search-engine-options = Search engine options
+mobile-compare-text-to-speech = Text to speech
+mobile-compare-reader-mode = Reader mode
+mobile-compare-spell-checking = Spell checking
+mobile-compare-web-extensionsaddons = Web extensions/Add-ons
+mobile-compare-dark-mode = Dark Mode
+mobile-compare-desktop-mode = Desktop Mode
+mobile-compare-find-on-page = Find on page
+mobile-compare-image-search = Image Search
+mobile-compare-voice-search = Voice Search
+mobile-compare-add-to-homescreen = Add to Homescreen
+mobile-compare-read-text-aloud = Read Text Aloud
+mobile-compare-news-feed = News Feed
+mobile-compare-collections = Collections
+mobile-compare-inapp-vpn = In-app VPN
+mobile-compare-shopping = Shopping
+mobile-compare-mobile-os-availability = Mobile OS availability
+mobile-compare-syncs-with-desktopmobile = { -brand-name-sync }s with desktop/mobile app
+mobile-compare-password-management = Password management
+mobile-compare-primary-password = Primary password
+
+mobile-compare-get-firefox-for-mobile = Get { -brand-name-firefox } for mobile
+mobile-compare-send-a-download-link-to-your = Send a download link to your phone.
+mobile-compare-scan-the-qr-code-to-get-started = Scan the QR code to get started
+mobile-compare-scan-alt-text = Scan the QR code to get { -brand-name-firefox } on your mobile device
+
```

Comment:
At the end of this page: https://www-dev.allizom.org/en-US/firefox/browsers/mobile/compare/ - there is a blurb on the versions being compared and how often this page might be updated. Are we intentionally omitting it for l10n?

Reply:
The blurb uses existing strings from firefox/browsers/compare/shared.ftl